salt/modules/rpmbuild.py

```diff
@@ -182,8 +191,8 @@ def make_src_pkg(dest_dir, spec, sources, env=None, template=None, saltenv='base
     for src in sources:
         _get_src(tree_base, src, saltenv)
 
-    # make source rpms for dist el5, usable with mock on other dists
-    cmd = 'rpmbuild --define "_topdir {0}" -bs --define "_source_filedigest_algorithm md5" --define "_binary_filedigest_algorithm md5" --define "dist .el5" {1}'.format(tree_base, spec_path)
+    # make source rpms for dist el6 with SHA256, usable with mock on other dists
+    cmd = 'rpmbuild --verbose --define "_topdir {0}" -bs --define "dist .el6" {1}'.format(tree_base, spec_path)
```

Comment:
@vutny Yes, as the new minimum supported, previously it was '.el5' as the minimum level.